xskillscore/tests/test_deterministic.py

```diff
@@ -130,20 +131,108 @@ def test_correlation_metrics_xr(a, b, dim, weight_bool, weights, metrics):
     assert_allclose(actual, expected)
 
 
-@pytest.mark.parametrize("metrics", distance_metrics)
+@pytest.mark.parametrize("metrics", correlation_metrics)
 @pytest.mark.parametrize("dim", AXES)
 @pytest.mark.parametrize("weight_bool", [True, False])
 @pytest.mark.parametrize("skipna", [True, False])
-@pytest.mark.parametrize("has_nan", [True, False])
-def test_distance_metrics_xr(a, b, dim, weight_bool, weights, metrics, skipna, has_nan):
+def test_correlation_metrics_xr_nan(
+    a_nan, b_nan, dim, weight_bool, weights, metrics, skipna
+):
+    """Test whether correlation metric for xarray functions (from
```

Comment:
I am doubting the purpose of these tests now. Aren’t we essentially only testing whether the xs.metric function does the same as the np_deterministic function? Is there are reason why this should not work? Isn’t this all independent of chunking and nans?